ruby_event_store/lib/ruby_event_store/mappers/transformation/encryption.rb

```diff
@@ -0,0 +1,125 @@
+module RubyEventStore
+  module Mappers
```

Comment:
Maybe `EncryptionKey` and `ForgottenData` could be here as well?

Reply:
The reason for it probably was:
- no change of already published (released) consts
- no divergence in class name and its location in filesystem structure